src/ibek/ioc.py

```diff
@@ -112,6 +122,10 @@ def lookup_instance(id):
     # it
     fields.append(("entity_enabled", bool, field(default=cast(Any, True))))
 
+    # add in the values fields as class attributes
+    for value in definition.values:
+        fields.append((value.name, str, field(default=cast(Any, value.value))))
+
```

Comment:
But this doesn't seem needed

Reply:
This was wrong because it was not using ClassVar (meaning that the value fields were interfering with the schema).
But I do think we need something here. This is where we generate an Entity Instance from a Definition (class-like thing) and we are responsible for constucting all the fields that end up in the Entity. values don't make it to the entity if we dont do this and indeed the calculated values of card_id fail to render in the test test_example_sr_rf_08 if I comment out the 'for value in definition.values ...'